source/includes/extracts-atlas-search-facts.yaml

```diff
@@ -0,0 +1,24 @@
+ref: fact-text-search-legacy-atlas
+content: |
+
+  MongoDB offers a :atlas:`premium full-text search solution, MongoDB 
+  Atlas Search </atlas-search/>`, for data hosted on :atlas:`MongoDB 
+  Atlas </>`. A legacy text search capability is available for users 
+  self-managing MongoDB deployments. 
+
+---
+ref: fact-regex-search-legacy-atlas
+content: |
+
+  MongoDB offers a :atlas:`premium full-text search solution, MongoDB 
+  Atlas Search </atlas-search/>`, for data hosted on :atlas:`MongoDB 
```

Comment:
Consider flipping the order of these phrases so users know immediately whether this is relevant to them.

"For data hosed on MongoDB Atlas, MongoDB offers..."